### PR TITLE
Add sway compositor and some add-ons

### PR DESCRIPTION
**Pull request by buianhhuy96**
- Add app-launchers module to store launcher information of default apps: chromium, terminal, nmLauncher
- Add nmLauncher and squeekboard as nixpkgs apps
- Add config for sway desktop
- Configure sway to map (rugged-7230) tablet touchscreen to its screen
- Add application profile to choose compositor
- Some features: taskbar, tray, launcher, clock, CPU and RAM Usage indicators (using nwg-panel), virtual keyboard for touchscreen (squeekboard)
- Minor issue: Chromium does not automatically trigger virtual inputs (other web browser do). But virtual keyboard can be popped up manually with a button on the taskbar
- Add custom libsecret with checkPhase disabled to bypass upstream error

### CODE REVIEW

**Comment by buianhhuy96** [comment]
I saw there was a new wallpaper for FMO-OS, in case @jsvapiav you want to update it

File: modules/graphics/sway/config [hunk 1]
```diff
@@ -0,0 +1,197 @@
+# Default config for sway
+#
+
+# Logo key. Use Mod1 for Alt.
+set $mod Mod4
```
Comment (jsvapiav) [inline modules/graphics/sway/config:5]: Let's change that to Mod1 due to screen keyboard does not have a Win Button?.

File: modules/graphics/sway/nwg-panel/config [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+    ],
+    "name": "panel-top",
+    "output": "All",
+    "padding-horizontal": 3,
+    "padding-vertical": 0,
+    "position": "top",
+    "scratchpad": {
+      "angle": 0.0,
+      "css-name": "Ghaf",
+      "icon-size": 16,
```
Comment (jsvapiav) [inline modules/graphics/sway/nwg-panel/config:75]: Maybe we need a little bit bigger panel due to it is pretty hard to push buttons on touch screen?

**Review by jsvapiav — APPROVED** [review]

File: modules/custom-packages/libsecret/default.nix [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+(final: _prev: {
+  libsecret = _prev.libsecret.overrideAttrs (old : {
+      doCheck = false;
```
Comment (jsvapiav) [inline modules/custom-packages/libsecret/default.nix:5]: Oh, one more, please add here comment: `WAR: {{war description}}`

**Review by jsvapiav — APPROVED** [review]

File: modules/graphics/sway/config (hunk 1 above)
Comment (buianhhuy96) [reply modules/graphics/sway/config:5]: yes, I left most sway config as default so any changes are welcome.

**Comment by buianhhuy96** [comment]
All the above comments are fixed. However, there are different squeekboard layouts. The one that is used in terminal has "alt" key but the normal one (when terminal closes) does not have "alt" key, so cannot be used to open terminal anyway.

For this, I think we can do a separate PR to improve squeekboard layout.

**Pull request merged**